package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@semantic-release/npm",
   "description": "semantic-release plugin to publish a npm package",
-  "version": "0.0.0-development",
+  "version": "1.0.0",
```

Comment:
we let semantic-release track the version with tags, so we intentionally do not maintain the version here. this is also not the correct version based on what has been released for this package